client/connect-account-page/index.tsx

```diff
@@ -74,6 +117,125 @@ const ConnectAccountPage: React.FC = () => {
 		return source;
 	};
 
+	const updateLoaderProgress = ( maxPercent: number, step: number ) => {
+		if ( loaderProgressRef.current < maxPercent ) {
+			const newProgress = loaderProgressRef.current + step;
+			setTestDriveLoaderProgress( newProgress );
+		}
+	};
+
+	const checkAccountStatus = () => {
+		// Fetch account status from the cache.
+		apiFetch( {
+			path: `/wc/v3/payments/accounts`,
+			method: 'GET',
+		} ).then( ( account ) => {
+			// Fake update loader progress bar percentage.
+			updateLoaderProgress( 100, 5 );
+
+			// If the account status is complete, redirect to the overview page.
+			// Otherwise, schedule another check after 5 seconds.
```

Comment:
Suggestion: if it takes longer than 5/10 seconds, maybe we should update the message on the loader? Something like "This is taking a bit longer than expected"? Maybe one to check with design/product to see if that is something they want, though.

Reply:
I discussed this with Orcun and Elizabeth, and they stated that the progress bar should clearly indicate the remaining time. @orcunomattic, do you think we should update the messages?

Reply:
I don’t think we need to update the message but animating the progress bar would be sufficient.